### PR TITLE
Fix for #183

### LINKED ISSUE #183: CopyAsHtml font-size property. Explicitly specify the unit of measurement - pixels.
CopyAsHtml calculate CSS property `font-size` in pixels, but it's not specifying strictly as `px`. So, some of html renders didn't understand it correctly. For example:
Current: 
`font-size:12` 
Need to be: 
`font-size:12px`

### PR DESCRIPTION
**Pull request by kirsan31**
Fix #183

### CODE REVIEW

File: src/CopyAsHtml/CopyAsHtml/HtmlMarkupProvider.cs [hunk 1]
```diff
@@ -329,7 +329,7 @@ private string GetCSS(IClassificationType classificationType)
             string currentSize = GetFontSize(properties);
             if (!string.IsNullOrEmpty(currentSize))
             {
-                styles.Add("font-size", currentSize);
+                styles.Add("font-size", currentSize + "px");
```
[the remark this answers is not in the document]
Comment (justcla) [reply src/CopyAsHtml/CopyAsHtml/HtmlMarkupProvider.cs]: I've checked. EM is wrong. The font size returned is a pixel unit, not em.

[the remark this answers is not in the document]
Comment (kirsan31) [reply src/CopyAsHtml/CopyAsHtml/HtmlMarkupProvider.cs]: Yea, px is correct. And yes, GetFontSize is far more better place, sorry.

File: src/CopyAsHtml/CopyAsHtml/HtmlMarkupProvider.cs [hunk 2]
```diff
@@ -329,7 +329,7 @@ private string GetCSS(IClassificationType classificationType)
             string currentSize = GetFontSize(properties);
```
[the remark this answers is not in the document]
Comment (kirsan31) [reply src/CopyAsHtml/CopyAsHtml/HtmlMarkupProvider.cs]: Done.

**Comment by kirsan31** [comment]
Sorry for late response,  I am back :) All done.

**Comment by justcla** [comment]
Perfect. :-)

**Comment by justcla** [comment]
Thanks @kirsan31 

I have merged te fix internally and sent it out for building.
We're just waiting for permission to sign and release.

**Comment by justcla** [comment]
@prnadago Can you please arranging signing of the build? Thanks

**Comment by prnadago** [comment]
Yup all signed and uploaded the latest vsix to Marketplace.
Thanks for your contribution @kirsan31!

**Pull request merged**